pkg/operator/ceph/object/health.go

```
@@ -149,8 +150,12 @@ func (c *bucketChecker) checkObjectStoreHealth() error {
 	s3SecretKey = *user.SecretKey
 
 	if c.objectStoreSpec.IsTLSEnabled() {
-		sslSecretCert, _ := c.context.Clientset.CoreV1().Secrets(c.namespacedName.Namespace).Get(ctx, c.objectStoreSpec.Gateway.SSLCertificateRef, metav1.GetOptions{})
-		sslCert = sslSecretCert.Data[certKeyName]
+		if c.objectStoreSpec.Gateway.SSLCertificateRef != "" {
+			sslSecretCert, _ := c.context.Clientset.CoreV1().Secrets(c.namespacedName.Namespace).Get(ctx, c.objectStoreSpec.Gateway.SSLCertificateRef, metav1.GetOptions{})
+			sslCert = sslSecretCert.Data[certKeyName]
+		} else if c.objectStoreSpec.GetServiceServingCert() != "" {
+			sslCert, _ = ioutil.ReadFile(ServiceServingCertCAFile)
```

Why ignoring the error from `ioutil.ReadFile`?